CHANGELOG.rst

```diff
@@ -2,6 +2,17 @@
 SDK Changelog
 =============
 
+2.5.0-alpha.0 (2024-10-23)
+==========================
+
+First preview release of the upcoming 2.5.0 version.
+
+* [#4544] Added design tokens for configuring the position of the previous page link.
+  You can now choose if the link should be at the top, the bottom or both. By default
+  this is shown at the bottom. The ability to add an icon has been added as well.
+* [#4398] Made it possible to cache initialDataReference. This was not passed properly
+  and it wouldn't be available in the submissionCreate.
```

Comment:
Can you apply the existing pattern for feature releases with the different sections?

```rst
**New features**

...

**Bugfixes**

...

```